modules/swagger-codegen/src/main/resources/csharp/api.mustache

```diff
@@ -81,10 +81,10 @@ namespace {{packageName}}.Api
         /// <returns></returns>
         public {{classname}}(String basePath)
         {
-            this.Configuration = new Configuration(new ApiClient(basePath));
+            this.Configuration = new {{packageName}}.Client.Configuration(new ApiClient(basePath));
```

Comment:
I made those changes and opened a PR to your branch. wing328/swagger-codegen#5

I didn't update the yaml (see my other comment).